code/modules/goonchat/browserassets/css/browserOutput.css

```diff
@@ -83,7 +83,7 @@ a.popt {text-decoration: none;}
 #ping {
 	position: fixed;
 	top: 0;
-	right: 40px;
+	right: 80px;
```

Comment:
I want to point out that this is super dumb, but I'm not very experienced in styling, and my attempts to make it properly follow the other elements didn't work.